### DISCUSSION TITLE
How to set pagination_next and pagination_prev using id in front matter

### DISCUSSION THREAD

**Discussion by awdhaan** [facebook/docusaurus#7063] [Q&A] [answered]
Does anybody know how I can use the pagination_next and pagination_prev front matter variables to link to a specific document id.

I have tried setting a specific document as follows:

`---
id: 1

title: some document
---`

and then linking that from another document as follows:

`---
title: some other document

pagination_next: null

pagination_prev: 1
---`

(original code does not contain blank lines)

when building the project, it errors out that the id cannot be found.

What should I be doing differently?

**Comment by Josh-Cena** [comment, marked as answer]
Hi! It's a bit confusing, but currently the `id` front matter only changes the _last part_ of a doc's ID. For example, if your doc is at `/docs/tutorial/basic/intro.md`, adding `id: 1` can only make its ID become `tutorial/basic/1`. You still need to use the full ID when using `pagination_prev`.

See #6018 where we also plan to make the `id` front matter declare the full ID instead of the last part.
